src/pages/publication/public_publication/PublicPublicationContent.tsx

```diff
@@ -139,7 +148,14 @@ const PublicPublicationContent: FC<PublicPublicationContentProps> = ({ publicati
           )}
           {publicationContext && (
             <LabelContentRow minimal multiple label={`${t('references.journal')}:`}>
-              <NormalText>{publicationContext.title}</NormalText>
+              <StyledContainer>
+                <NormalText>{publicationContext.title}</NormalText>
+                {publicationContext?.url && (
```

Comment:
```suggestion
                {publicationContext.url && (
```
vil i så fall krasja på linja over